src/lib/components/utils/TagList.svelte

```diff
@@ -32,11 +32,11 @@
     target: 'popupAutocomplete',
     placement: 'bottom-start'
   };
-
   let tagList = [];
-  const loadTagList = () => (tagList = tags.map((t: Tag) => t.name));
+  const loadTagList = tags ? () => (tagList = tags.map((t: Tag) => t.name)) : () => {};
```

Comment:
`tags` is initialized to `[]`, so the else case of this conditional is never taken